packages/components/src/RecurringSelect/RecurringSelect.mdx

```diff
@@ -0,0 +1,64 @@
+---
+name: RecurringSelect
+menu: Components
+route: /components/recuringselect
+showDirectoryLink: true
+---
+
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { RecurringSelect } from "./RecurringSelect";
+import { Content } from "@jobber/components/Content";
+import { Text } from "@jobber/components/Text";
+import { Checkbox } from "@jobber/components/Checkbox";
+import { Divider } from "@jobber/components/Divider";
+import { Avatar } from "@jobber/components/Avatar";
+import { useState } from "react";
+import { DurationPeriod, SchedulerState } from "./types";
+import { useScheduler } from "./useScheduler";
+
+# RecurringSelect
+
+<ComponentStatus stage="ready" responsive="yes" accessible="yes" />
+
+A `RecurringSelect` is used to provide an intuitive, accessible pre-built UI
```

Comment:
@chriscarline if you could flesh out the rest of the sections here (adding more "when to use / when not to use", Related Components, etc, that would be 😗 👌 - could also happen in a followup PR